package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "private": true,
+  "author": "18F",
+  "license": "CC0-1.0",
+  "scripts": {
```

Comment:
I'm assuming these are a convenience for devs of the guides style gem to periodically update the standards, and not for owners of sites? If so :+1: